src/app/pages/storage/modules/pool-manager/utils/generate-vdevs/generate-vdevs-treat-as-minimum.spec.ts

```diff
@@ -97,15 +94,9 @@ describe('GenerateVdevsService - treat disk size as minimum', () => {
       });
 
       expect(vdevs).toEqual({
-        [VdevType.Data]: expectDisks([
-          ['enclosure1-disk1', 'enclosure1-disk2', 'enclosure1-disk3'],
-        ]),
-        [VdevType.Log]: expectDisks([
-          ['small-ssd1'],
-        ]),
-        [VdevType.Spare]: expectDisks([
-          ['small-ssd2'],
-        ]),
+        [VdevType.Data]: expectDisks([['enclosure1-disk1', 'enclosure1-disk2', 'enclosure1-disk3']]),
+        [VdevType.Log]: expectDisks([['enclosure2-never-used-too-small']]),
```

Comment:
Let's do something with `enclosure2-never-used-too-small`. It's name doesn't make sense anymore . It can probably be removed.